packages/swapper/src/types.ts

```diff
@@ -345,6 +347,7 @@ export type EvmTransactionRequest = {
 
 export type CowMessageToSign = {
   chainId: ChainId
+  from: string
```

Comment:
not seeing this used anywhere

Reply:
nice catch! Legacy from first fiddlings with this - 167b858012